docs/dev-guide/contributing.md

```diff
@@ -3,25 +3,49 @@ id: dev-guide-contributing
 title: Contributing Guidelines
 ---
 
-# How to contribute
-We would love to have your help. Before you start working however, please read
-and follow this short guide.
+# How to Contribute 🤝
+We greatly appreciate your willingness to contribute. Before you start working however, please take a moment to read and follow this brief guide.
 
-# Reporting Issues and Asking Questions
+# Reporting Issues and Asking Questions 💭
 
 We prefer issues to be discussed first in the [community forum](https://community.jitsi.org/) and when confirmed, then an issue can be opened in the [issue tracker](https://github.com/jitsi/jitsi-meet/issues).
 
-When opening an issue, please provide as much information as possible. Mention the version of Jitsi Meet, Jicofo and JVB you are using, and explain (as detailed as you can) how the problem can be reproduced.
+Feel free to report ***any bugs, ask for new features or anything else*** you need help with. When opening an issue, please provide as much information as possible.
 
-Please ask any general and implementation specific questions on the [community forum](https://community.jitsi.org/) for support.
+For bugs please follow these steps:
```

Comment:
We ask people to report and discuss problems in the community forum, and when we think that is an issue with the code we ask them to open a ticket in the relevant repo. 

Many times they create tickets which we close as they are in wrong repo, or that is not a code problem, but configuration problems, or those are issues they have with meet.jit.si and so on.
This step is important and should not be removed.

Reply:
I have this sentence that explains it. But if it is not very clear, I can add back what you sent

<img width="1208" alt="Screenshot 2024-05-02 at 14 41 32" src="https://github.com/jitsi/handbook/assets/43909097/479c3f60-8e20-43c0-971f-5958bba900bf">

Reply:
The first step of reporting an issue is opening a discussion in the forum.

Reply:
Okay I edited it and made a few more edits